js/components/common/react_modal.js.jsx

```diff
@@ -23,41 +23,36 @@ class CourseModal extends React.Component {
       courseId: '',
       course: [],
       sessions: [],
-      modalIsOpen: false,
       courseTitle: '',
     };
-    this.openModal = this.openModal.bind(this);
     this.closeModal = this.closeModal.bind(this);
   }
 
-  openModal(newCourse) {
-    if (newCourse === this.state.courseId) {
-      this.setState({ modalIsOpen: true });
-    } else {
-      getCourse(newCourse)
+  closeModal() {
```

Comment:
This function isn't necessary to define. You can just past `this.props.onClose` directly to the `ReactModal` component.